openadapt/window/_windows.py

```diff
@@ -149,6 +201,34 @@ def dictify_rect(rect: pywinauto.win32structures.RECT) -> dict:
     return rect_dict
 
 
+def get_element_properties_pgw(window: gw.Window) -> dict:
+    """Simulates retrieving properties of a window element and its children.
+
+    Args:
+        window: An instance of a pygetwindow Window.
+
+    Returns:
+        dict: A nested dictionary containing the properties of the window.
+        The dictionary includes a "children" key, which will be an empty list
+        since pygetwindow does not support child elements retrieval.
+    """
+    properties = {
+        "title": window.title,
+        "left": window.left,
+        "top": window.top,
+        "width": window.width,
+        "height": window.height,
+        "rectangle": {
+            "left": window.left,
+            "top": window.top,
+            "width": window.width,
+            "height": window.height,
+        },
+        "children": [],  # pygetwindow does not support children
```

Comment:
I think if pygetwindow doesn't support children, then we don't need this function.

The purpose of pygetwindow is to get the active window with as little latency as possible. In addition to this, we also need to be able to get the full window accessibility data.